src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import useToggle from '#hooks/useToggle';
+import data from './recommendations.ltr.json';
+
+const CpsRecommendations = () => {
+  const { recommendations } = useContext(ServiceContext);
+  const { enabled } = useToggle('cpsRecommendations');
+
+  const { hasStoryRecommendations } = recommendations;
+
+  if (!hasStoryRecommendations || !enabled) return null;
+
+  return <div>Recommendations Container</div>;
+};
+
+export default CpsRecommendations;
+
+CpsRecommendations.propTypes = {
+  /*
+  items: arrayOf(
+    shape({
+      assetUri: string.isRequired,
+      shortHeadline: string.isRequired,
+      imageHref: string.isRequired,
+    }),
+  ),
+  parentColumns: shape({
+    group0: number,
+    group1: number,
+    group2: number,
+    group3: number,
+    group4: number,
+    group5: number,
+  }).isRequired,
+  */
+};
+
+CpsRecommendations.defaultProps = {
+  items: data.items,
+};
```

Comment:
Would it make sense to render something temporary using these props? They're going to be part of the API moving forward and I feel they we should be validating them in the current snapshot test.

Reply:
@simonsinclair 

> Would it make sense to render something temporary using data from these props? They're going to be part of the API moving forward and it might be worth validating what's being passed from the StoryPage in the current tests.

Nothing is being passed in from the story page atm as data fetching hasn't been added yet, we're mostly defaulting to use the mundo fixture as part off this build. I could render a list of the headline text though if that's okay.

Reply:
Thanks, Rowland – as discussed in the call this morning, that would be great.